lib/wallet/src/Cardano/Wallet/Write/Tx/Balance.hs

```diff
@@ -159,6 +159,7 @@ import qualified Cardano.Wallet.Primitive.Types.UTxO as W
 import qualified Cardano.Wallet.Primitive.Types.UTxO as UTxO
 import qualified Cardano.Wallet.Primitive.Types.UTxOIndex as UTxOIndex
 import qualified Cardano.Wallet.Primitive.Types.UTxOSelection as UTxOSelection
+import qualified Cardano.Wallet.Write.Tx as Write.Tx
```

Comment:
This import already exists as `WriteTx`
```suggestion
```

Since Yura also seemed to default to this, I added this topic to next Monday's meeting, but could we stick to `WriteTx` for now?

Reply:
Added to the next meeting minutes here:
https://docs.google.com/document/d/1aYMejoSxuE6VzHvqT29j9c0vuphzT0-QsHJo_PMFTRc/edit#